src/ProgressOnderwijsUtils/Data/SqlBatch.cs

```diff
@@ -58,6 +58,23 @@ public void Execute(SqlConnection conn, out int nrOfRowsAffected)
     }
 }
 
+public readonly record struct DbColumnSchemaCommand(ParameterizedSql Sql, CommandTimeout CommandTimeout) : ITypedSqlCommand<DbColumn[], DbColumnSchemaCommand>
+{
+    public DbColumnSchemaCommand WithTimeout(CommandTimeout timeout)
+        => this with { CommandTimeout = timeout, };
+
+    public DbColumn[] Execute(SqlConnection conn)
+    {
+        using var cmd = this.ReusableCommand(conn);
```

Comment:
De conn is dan al op de juiste user gezet toch? Tenminste bij ons gebruik hiervan straks in de pnet app.

Reply:
Dat ligt dan aan de afnemer, maar ja, dat is wel de intentie natuurlijk.